libyui-rest-api/src/YDumbTabActionHandler.cc

```diff
@@ -0,0 +1,61 @@
+/*
+  Copyright (C) 2021 SUSE LLC
+
+  This library is free software; you can redistribute it and/or modify
+  it under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation; either version 2.1 of the
+  License, or (at your option) version 3.0 of the License. This library
+  is distributed in the hope that it will be useful, but WITHOUT ANY
+  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+  FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+  License for more details. You should have received a copy of the GNU
+  Lesser General Public License along with this library; if not, write
+  to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+  Floor, Boston, MA 02110-1301 USA
+*/
+
+#include <boost/algorithm/string.hpp>
+
+#define YUILogComponent "rest-api"
+#include <yui/YUILog.h>
+
+#include <yui/YDumbTab.h>
+#include <yui/YItem.h>
+
+#include "YDumbTabActionHandler.h"
+
+
+std::function<void (YDumbTab*)> YDumbTabActionHandler::get_handler( YDumbTab * widget,
+                                                                    const std::string &itemLabel )
+{
+    return [&] (YDumbTab *tab) {
+        YItem * item = findItemOmittingAmpersand( widget, itemLabel );
+        if ( item )
+        {
+            yuiMilestone() << "Activating Tree Item \"" << item->label() << '"' << std::endl;
+            tab->setKeyboardFocus();
+            tab->selectItem( item );
+            activate_widget( tab );
+        }
+        else
+        {
+            throw YUIException("Item: '" + itemLabel + "' cannot be found among tabs");
+        }
+    };
+}
+
+YItem * 
+YDumbTabActionHandler::findItemOmittingAmpersand( YDumbTab * widget, const std::string &itemLabel ) const
+{
+    for ( YItemConstIterator it = widget->itemsBegin(); it != widget->itemsEnd(); ++it )
+    {
+        YItem * item = *it;
+
+        if ( boost::erase_all_copy(item->label(), "&") == itemLabel )
+        {
+            return item;
+        }
+    }
+
+    return 0;
```

Comment:
I was also thinking of what is more conventional in this case, and found out that `0` is used in YSelectionWidget.cc https://github.com/libyui/libyui/blob/dabdcd9cc6a0058fd6966d7d2e19d508debcc0ac/libyui/src/YSelectionWidget.cc#L541. 
I'll change this to `nullptr`, though. I guess it will not harm.

Reply:
Done.